src/sidebar/components/MarkdownEditor.js

```diff
@@ -138,20 +144,52 @@ function ToolbarButton({
 
   if (label) {
     return (
-      <LabeledButton classes="font-normal TransparentButton" {...buttonProps}>
+      <LabeledButton
+        classes={classNames(
+          'font-normal bg-transparent',
+          // TODO: Refactor shared button styles to reduce specificity and make
+          // this !important rule unnecessary
+          'hover:!bg-transparent'
+        )}
+        {...buttonProps}
+      >
         {label}
       </LabeledButton>
     );
   }
   return (
-    <IconButton classes="MarkdownEditor__toolbar-button" {...buttonProps} />
+    <IconButton
+      classes="py-3.5 px-2 text-tiny touch:text-base"
+      {...buttonProps}
+    />
   );
 }
 
 /**
  * @typedef {'bold'|'italic'|'quote'|'link'|'image'|'math'|'numlist'|'list'|'preview'|'help'} ButtonID
  */
 
+/**
+ * @typedef {import('preact').Ref<HTMLTextAreaElement>} TextAreaRef
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLTextAreaElement>} TextAreaAttributes
+ *
+ * @param {TextAreaAttributes & { classes?: string, containerRef?: TextAreaRef }} props
+ */
+function TextArea({ classes, containerRef, ...restProps }) {
```

Comment:
This is the only `textarea` element in the `client` application but I'm treating this as an extractable presentational component. If we keep on like this I might want to bring up the topic of forwarding refs again, as it's a bit of a PITA to do it by hand.